safeeyes/safeeyes/config/lang/hu.json

```diff
@@ -0,0 +1,40 @@
+{
+    "meta_info": {
+        "language_name": "Magyar",
+        "language_name_en": "Hungarian"
+    },
+    "exercises": {
+        "long_break_exercises": [
+            "Sétáljon egyet!",
+            "Dőljön hátra és pihenjen!",
+            "Nyújtózzon eggyet!"
+        ],
+        "short_break_exercises": [
+            "Csukja be hosszabb időre a szemét!",
+            "Igyon egy kis vizet!",
+            "Forgassa meg a fejét becsukott szemmel!",
+            "Nyújtózzon eggyet!"
```

Comment:
@adamos42 Please note that the last short_break_exercise and last long_break_exercise are same. Could you please remove the duplicate.
Thank you for making Safe Eyes better :-)